xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/delete.vm

```diff
@@ -238,10 +238,30 @@
 #######################################################
 #macro(displayConfirmMessage)
   #if($xwiki.hasRecycleBin())
-    #if($hasInlinks)
-      <p><strong>$services.localization.render('core.recyclebin.confirmWithInlinks')</strong></p>
+    #if($xwiki.canSkipRecycleBin())
+      <div class="xform">
+        <dl>
+          <dt>
+            $services.localization.render('core.recyclebin.wantToSkip')
```

Comment:
From what I understand those translations are not meant to contain HTML so you should escape them to be safe.

Reply:
Is it ok to add `$escapetool.xml(...)` around other calls to `$services.localization.render(...)` of `delete.vm` where it is missing?

Reply:
Depends if HTML is expected from those translations (which is sometime the case).

Reply:
Ok, I did it for the labels of the displayConfirmMessage macro only